README.md

```
@@ -189,47 +158,280 @@ const signature = await walletClient.signMessage({
+
+### 🔍 Advanced Topics
+- **[Best Practices](./sdk/typescript/docs/best-practices.md)** - Development guidelines
+- **[Examples](./sdk/typescript/docs/examples-basic.md)** - Code samples
+- **[Types & Interfaces](./sdk/typescript/docs/types-interfaces.md)** - Type definitions
+
+## 🛡️ Security Features
+
+### Multi-Phase Security Model
+
+**Time-Delay Workflow:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)